### ISSUE TITLE
Collaboration Bug in anonymous style

### ISSUE THREAD

**Issue by augustfly** [AASJournals/AASTeX60#100]
This is a public bug report. 

Due to a bug in v6.3 authors must end with a \collaboration or \nocollaboration command in order for the author list to be hidden when the _anonymous_ option is used.

**Comment by augustfly** [comment]
@gregschwarz I don't understand this text of the bug report -- I think something is missing in this block of text copied from the Author Guide. 

Does it mean that all anonymized papers need a \collaboration or a \nocollaboration for anonymous to work? That's a bad enough bug we should fix it immediately.

**Comment by gregschwarz** [comment]
Yes, that is unfortunately the problem. A \nocollaboration{*} where "*" is the number of authors is required to have it work properly. When testing v6.3 I only used examples with the \collaboration and \nocollaboration  commands and did not realize that either was needed to make anonymous work.  On top of that I just today discovered that if you use the style command "longauthor" only the names are removed and not the affiliations.

**Comment by augustfly** [comment]
Yeah, I know we have zero bandwidth but I wonder if Amy can't try to figure this all out in the interim. Esp given how valuable 'anonymous' style will/may become.
